src/parser.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash'
```

Comment:
even if you sure lodash is good to have here, give me only the methods we need. 
`import _f1 from 'lodash/f1`